eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java

```diff
@@ -305,8 +310,10 @@ private boolean checkIfExistingRSOrCandidateForMerge(SubmittedVariantEntity subm
 
         if (allExistingHashesInDB.containsKey(variantHash)) {
             if (mergeCandidateSVOE.containsKey(variantHash)) {
-                mergeCandidateSVOE.get(variantHash).getInactiveObjects()
+                SubmittedVariantOperationEntity submittedVariantOperationEntity = mergeCandidateSVOE.get(variantHash);
+                submittedVariantOperationEntity.getInactiveObjects()
                         .add(new SubmittedVariantInactiveEntity(submittedVariantEntity));
+                updateMergeCandidateSVOE.put(variantHash, submittedVariantOperationEntity);
```

Comment:
This is present in the if and else statements, can be taken out so it is not repeated